src/main/java/software/amazon/encryption/s3/internal/ConvertSDKRequests.java

```diff
@@ -140,6 +143,70 @@ public static CreateMultipartUploadRequest convert(PutObjectRequest request) {
       .build();
   }
 
+  public static PutObjectResponse convertResponse(CompleteMultipartUploadResponse response) {
+    final PutObjectResponse.Builder output = PutObjectResponse.builder();
+    response
+            .toBuilder()
+            .sdkFields()
+            .forEach(f -> {
+              final Object value = f.getValueOrDefault(response);
+              if (value != null) {
+                switch (f.memberName()) {
+                  case "ETag":
+                    output.eTag((String) value);
+                    break;
+                  case "Expiration":
+                    output.expiration((String) value);
+                    break;
+                  case "ChecksumCRC32":
+                    output.checksumCRC32((String) value);
+                    break;
+                  case "ChecksumCRC32C":
+                    output.checksumCRC32C((String) value);
+                    break;
+                  case "ChecksumCRC64NVME":
+                    output.checksumCRC64NVME((String) value);
+                    break;
+                  case "ChecksumSHA1":
+                    output.checksumSHA1((String) value);
+                    break;
+                  case "ChecksumSHA256":
+                    output.checksumSHA256((String) value);
+                    break;
+                  case "ChecksumType":
+                    output.checksumType((String) value);
+                    break;
+                  case "ServerSideEncryption":
+                    output.serverSideEncryption((String) value);
+                    break;
+                  case "VersionId":
+                    output.versionId((String) value);
+                    break;
+                  case "SSEKMSKeyId":
+                    output.ssekmsKeyId((String) value);
+                    break;
+                  case "BucketKeyEnabled":
+                    output.bucketKeyEnabled((Boolean) value);
+                    break;
+                  case "RequestCharged":
+                    output.requestCharged((String) value);
+                    break;
+                  // Ignored fields: Location, Bucket, Key
+                  case "Location":
+                  case "Bucket":
+                  case "Key":
+                    // These fields exist only in CompleteMultipartUploadResponse, not in PutObjectResponse
+                    break;
+                  default:
+                    // We should silently drop unknown fields because,
+                    // once the object is stored we expect to return success response.
```

Comment:
Agree that we should not throw an exception, but we should emit a log info for awareness: 

```suggestion
                    // We should NOT throw an exception for unknown fields because
                    // once the object is stored, we expect to return a successful response.
                    // Emit a log at info level for awareness.
                    LogFactory.getLog(ConvertSDKRequests.class).info(f.memberName() + " returned in CompleteMultipartUploadResponse for "
                            + response.key() + " is an unknown field." +
                            "The S3 Encryption Client does not recognize this option and cannot set it on the CompleteMultipartUploadResponse." +
                            "This may be a new S3 feature." +
                            "Please report this to the Amazon S3 Encryption Client for Java: " +
                            "https://github.com/aws/amazon-s3-encryption-client-java/issues."
                    );
```

(also requires `import org.apache.commons.logging.LogFactory;`)